src/qibo/backends/qulacs.py

```diff
@@ -0,0 +1,87 @@
+import re
+
+import numpy as np
+import qulacs  # pylint: disable=import-error
+from qulacs import (  # pylint: disable=no-name-in-module, import-error
```

Comment:
Do we need this exceptions because otherwise pylint fails for mac with py3.9 in the CI? If yes, I guess that's fine, otherwise I am not sure if they are needed given that CI is using the test dependencies which include qulacs.

Reply:
Yes, if I remember correctly `pylint` was complaining in `macos-python3.9`

Reply:
I'd say this is definitely the best way you could do it. At least is confined within the `qibo.backends.qulacs` module, though not restricted to the offending macos+py3.9 combo.

If done in the workflow, it would be positioned in a larger scope, though you may be able to ignore specifically that file for that combo.